### ISSUE TITLE
Allow users to Request known data

### ISSUE THREAD

**Issue by leomira** [vexl-it/vexl#98]
We already have a design for this, but it was deferred due to the time pressure before ChainCamp.

The idea is having an in-app option for our users to request the data we know about him. 

I imagine it to be a simple dump of encrypted data from a DB in a form of downloadable pdf.

**Comment by slush0** [comment]
I suppose we won't have much of human readable stuff, right? What's the use case for this? We even don't know how many offers any user has...

**Comment by prusnak** [comment]
I don't think we need this in the app. They can always request data via email, but it will be encrypted mess anyway.
